features/functests/performance/performance.go

```diff
@@ -12,36 +13,47 @@ import (
 	"github.com/openshift-kni/baremetal-deploy/features/functests/utils/clients"
 	"github.com/openshift-kni/baremetal-deploy/features/functests/utils/namespace"
 
+	ocv1 "github.com/openshift/api/config/v1"
 	k8sv1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-const testNamespace = "perfomancetest"
+const (
+	perfTestNamespace            = "perfomancetest"
+	perfWorkerNodesLabel         = "node-role.kubernetes.io/worker-rt="
+	perfWorkerRtKernelMcTemplate = "11-worker-rt-kernel"
```

Comment:
youll  need to align to the changes introduced in  https://github.com/openshift-kni/baremetal-deploy/pull/115